hackclub.com.yaml

```
@@ -3222,6 +3222,10 @@ tbilisi:
   - ttl: 600
     type: CNAME
     value: tbilisihc.github.io.
+_github-pages-challenge-tbilisihc.tbilisi:
+  - ttl: 600
+    type: CNAME
+    value: 352ce0458af34bcb3ac0d2f8c7920d.
```

Is this what you meant?
```suggestion
    type: TXT
    value: 352ce0458af34bcb3ac0d2f8c7920d
```